lib/generator/src/Generator/AbstractTypeGenerator.php

```diff
@@ -226,15 +233,19 @@ protected function callableCallbackFromArrayValue(array $value, string $key, ?st
                     $compilerNames = $matches[1] ?? [];
                 }
             }
+
+            $extraCode = $this->generateExtraCode($value, $key, $argDefinitions, $default, $compilerNames);
```

Comment:
This is where the validation code will be generated and inserted in the resolver callback.